kubernetes-elasticsearch-cluster/create-cluster.sh

```diff
@@ -0,0 +1,57 @@
+#!/usr/bin/env bash
```

Comment:
```
#!/bin/bash
```
First, this matches existing scripts. Second, I can add `-x` to the end of `#!/bin/bash`.

Same for rest of PR.

Reply:
Done.